dashboard/config/linux/upstream-kmsan-base.config

```diff
@@ -5,12 +5,12 @@
 # Automatically generated file; DO NOT EDIT.
 # Linux/x86_64 5.18.0-rc4 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="clang version 15.0.0 (https://github.com/llvm/llvm-project.git 610139d2d9ce6746b3c617fb3e2f7886272d26ff)"
+CONFIG_CC_VERSION_TEXT="clang version 14.0.0 (https://github.com/llvm/llvm-project.git 38e16e1cebb891ad47b85727bf46f7dac6d7da94)"
```

Comment:
`syz-env make configs` is doing some changes to KMSAN config. Are these OK?

Reply:
ping

Reply:
15.0.0 is a clang-kmsan version introduced by [this commit](https://github.com/google/syzkaller/commit/8f633d840e3eb6454f036e9da3285bcf27345616)
One more detail - syz-big-env still have:
```
/usr/local/clang-kmsan/bin/clang --version
clang version 14.0.0 (https://github.com/llvm/llvm-project.git 38e16e1cebb891ad47b85727bf46f7dac6d7da94)
```

Reply:
@dvyukov , did you use syz-big-env?

Reply:
I used `syz-env`, but did not pull it (had an older version locally).
Updated now and the kmsan config diff went away. So everything is good.
PTAL